addons/website/views/snippets/s_form_aside.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_form_aside" name="Form Aside">
+    <section class="s_form_aside pt64 pb64">
+        <div class="container">
+            <div class="row">
+                <div class="col-12 col-lg-6 order-lg-0 pt-0 pb-0" style="order: 1;">
```

Comment:
Do we need these `pt-0` and `pb-0` ?

I find it weird that the system applies an utility for `order-lg` and an inner style for the other.